src/RemoteInboxNotifications/README.md

```diff
@@ -497,3 +497,17 @@ WooCommerce Admin plugin.
 ```
 
 No other values are needed.
+
+## Debugging Specification
+
+You can see the evaluation of each specification by turning on an optional evaluation logger.
+
+1. Define `WC_DEBUG_REMOTE_INBOX_NOTIFICATIONS` constant in `wp-config.php`. `define('WC_DEBUG_REMOTE_INBOX_NOTIFICATIONS', true);`
```

Comment:
As is, I think we may want to prefix this with `WC_ADMIN_`.

Another thought depending on where this constant will be used: we may want to make it something more generic like `WC_ADMIN_DEBUG_RULE_EVALUATOR` since the rule evaluations can also be used in the remote extensions and payment gateway suggestions now.

But we could optionally keep them separate if you feel that's advantageous and if the constant logic is removed from the `EvaluationLogger` class noted above.

Reply:
Thank you for the suggestion 👍 I like `WC_ADMIN_DEBUG_RULE_EVALUATOR`. It makes more sense. I will make an update.

Reply:
Now that I think about it..I think it might be better to separate them as we most likely work on them separately. I'm going to rename it to `WC_ADMIN_DEBUG_REMOTE_INBOX_NOTIFICATIONS`. What do you think?

Reply:
updated in https://github.com/woocommerce/woocommerce-admin/pull/7194/commits/1ae3aec150bed893c2311396a2039dc328710fd9

Reply:
I'd still love to see this as something more general.  If we choose to separate them, we'll have 3 different sets of constants and code for loggers.

We currently use rules in the inbox notifications, free extensions, and payment gateway suggestions.  This list may grow in the future.

Reply:
@joshuatf Sounds good. I'll make a change.